src/Hazelcast.Net/Exceptions/ExceptionMessages.cs

```diff
@@ -19,6 +19,11 @@ namespace Hazelcast.Exceptions
     /// </summary>
     internal static class ExceptionMessages
     {
+        /// <summary>
+        /// Internal error.
+        /// </summary>
+        public const string InternalError = "Internal error.";
```

Comment:
why did we need it? What is the purpose? I don't see it in `ClientProtocolErrorCodes.java`.

Reply:
why don't you use `assert` for such cases?

Reply:
Seems like .NET `Debug.Assert` just prints a message and does not abort the process, hence it is not preferred in C#.